content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -736,15 +736,21 @@ for more information.
 
 ### SecurityContextDeny {#securitycontextdeny}
 
-{{< feature-state for_k8s_version="v1.0" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.27" state="deprecated" >}}
 
 {{< caution >}}
-This admission controller plugin is **outdated** and **incomplete**, it may be
-unusable or not do what you would expect. It was originally designed to prevent
-the use of some, but not all, security-sensitive fields. Indeed, fields like
-`privileged`, were not filtered at creation and the plugin was not updated with
-the most recent fields, and new APIs like the `ephemeralContainers` field for a
-Pod.
+The Kubernetes project recommends that you **do not use** the
+`SecurityContextDeny` admission controller.
```

Comment:
Should we mention it is going to get removed in a future version of the project?

Reply:
Thanks for the review, I added this mention, please feel free to `/lgtm` since pushing removed the label.

Reply:
Added here https://github.com/kubernetes/website/pull/40166/files#diff-0e59d6cfd3e5333b3a5effab53d20c05b86016ccda4bc12a37077304924b523eR746